### ISSUE TITLE
If support UIView？

### ISSUE THREAD

**Issue by wy19901227** [hackiftekhar/IQKeyboardManager#455]
HI, If I Write a textView base on UIView，So，When MyTextView becomFirstRespond,How Can I make it can be Know By IQKeyBadrdManager?

**Comment by hackiftekhar** [comment]
If you are writing your own textView using UIView and CoreText framework then I need to check if it will work with IQKeyboardManager. I'm not 100% sure.

**Comment by wy19901227** [comment]
I did check ,IQ will not identify it as a textView

**Comment by hackiftekhar** [comment]
Then you can share a demo project with me and I'll find-out a solution for you.

**Comment by wy19901227** [comment]
[YYTextViewDemo.zip](https://github.com/hackiftekhar/IQKeyboardManager/files/177340/YYTextViewDemo.zip)

**Comment by wy19901227** [comment]
Thank you.

**Comment by hackiftekhar** [comment]
I checked the demo and I found two reasons with YYTextView.

1) YYTextView is firing `YYTextViewTextDid....` notifications while IQKeyboardManager is listening only for `UITextViewTextDid....` notifications. That's why IQKeyboardManager is unable to detect YYTextView.

2) When I replaced `YYTextViewTextDid....` notifications with `UITExtViewTextDid....` notifications, then I got an issue with the toolbar with Done button. Done button doesn't appear at first time. So probably we need to 'reloadInputViews' manually.

Let me know what you thought about this.
